lib/BackgroundJob/ExpireGroupVersions.php

```diff
@@ -33,6 +33,7 @@ class ExpireGroupVersions extends \OC\BackgroundJob\TimedJob {
 	public function __construct(GroupVersionsExpireManager $expireManager) {
 		// Run once per hour
 		$this->setInterval(60 * 60);
+		$this->setTimeSensitivity(self::TIME_INSENSITIVE);
```

Comment:
hmm, now I see that we didn't make the regular version expiry time insensitive: https://github.com/nextcloud/server/blob/master/apps/files_versions/lib/BackgroundJob/ExpireVersions.php#L52

and since this was set to run every hour I'm not sure about the impact if we have expiration only running during the night

@nickvergessen what were your thoughts when leaving the version expiration unchanged in core ?

Reply:
Well if the job should run once an hour it is not suitable for this. if it runs only in maintenance time, it could run once a day only as well or is it doing junking? then it could run all the time in maintenance window?

I skipped the things in core when the time was not "once a day" as that seemed to not be what we where looking for.

Reply:
alright, thanks for confirming. let's close this